it/it-tests/src/test/java/it/authorisation/AuthenticationTest.java

```diff
@@ -89,9 +91,9 @@ public void basic_authentication_based_on_login_and_password() {
     createUser(login, name, password);
 
     // authenticate
-    WsClient wsClient = new WsClient(newHttpConnector().url(ORCHESTRATOR.getServer().getUrl()).login(login).password(password).build());
```

Comment:
- I'd prefer a static method than `HttpConnector.Builder()`, maybe newHttpConnectorBuilder()
- +1 for replacing `login(string)` and `passowrd(string)` by `credentials(string, string)`
- as discussed, I'd set the url directly in the http connector constructor directly, as a mandatory field. Out of scope of this PR